generated/nifake/nifake/unit_tests/test_session.py

```diff
@@ -1308,12 +1319,12 @@ def test_import_attribute_configuration_buffer_list_i8_big(self):
                 try:
                     session.import_attribute_configuration_buffer(configuration)
                     assert False
-                except OverflowError:
+                except ValueError:
                     pass
 
-    def test_import_attribute_configuration_buffer_list_i8_big_float(self):
+    def test_import_attribute_configuration_buffer_list_i8_float(self):
```

Comment:
I question whether we should have explicit tests for calling `import ` with a list of floats. It seems like a very invalid use case even if it happens to work due to duck typing.